test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -88,6 +90,12 @@ static void test(Path top) throws IOException {
                 supportsCreationTimeRead = true;
                 supportsCreationTimeWrite = true;
             }
+        } else if (Platform.isLinux()) {
+            // Creation time read depends on statx system call support
+            supportsCreationTimeRead = Linker.nativeLinker().defaultLookup().find("statx").isPresent();
+            System.out.println("[Linux] Debug: supportsCreationTimeRead == " + supportsCreationTimeRead);
```

Comment:
I think you left a debug statement here, or if you didn't then maybe drop the work "Debug" as it looks out of place here.

Reply:
If you do leave a trace message here then better to put it after the if-then-else so that it's clear on all platforms whether file create time is tested or not.

Reply:
OK.